actions/common.go

```diff
@@ -120,3 +130,11 @@ func getFullPath(dirName string) string {
 	}
 	return filepath.Join(currentWorkingDir, dirName)
 }
+
+func getAllGitRepoNames(appended []string, toAppend []string) []string {
+	ret := appended
+	for _, repoName := range toAppend {
+		ret = append(ret, repoName)
+	}
+	return ret
+}
```

Comment:
See [comment](https://github.com/deis/deisrel/pull/74/files#r65606310) above. You can get by without this function.